_quarto.yml

```diff
@@ -2,6 +2,9 @@ project:
   title: "LaplaceRedux.jl"
   execute-dir: project
 
+render:
```

Comment:
@pat-alt  i tried placing this like piece of code both here and in  /docs/_metadata.yml but it doesn't work. if i run quarto render . from inside the /tutorials folder i get 44 new files.

Reply:
I think this needs to be indented to fall under the project settings:

```yaml
project:
  title: "{{{PKG}}}.jl"
  execute-dir: project
  render:
    - "!docs/src/*.md"
```

Reply:
@pat-alt no, it stops working completely if i do this. btw how do you post code blocks in the comments?

Reply:
Just markdown syntax https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
What do you mean by "it stops working completely"?

Reply:
nvm found the solution

Reply:
@pat-alt the documentation says that when you use the prefix ! "you need to start by specifying everything you do want to render. For example: project:
  render:
    - "*.qmd"
    - "!ignored.qmd"
    - "!ignored-dir/"

Reply:
You're right, thanks!

Reply:
@pat-alt  is there a way to preview the website? i would like to check if everything is okay but it doesn't seems to be a command. Seems like i am missing a step... once i render the .qmd files, how do i rerender the html part?

Reply:
> This is handled by Documenter.jl, simply run `julia --project=docs docs/make.jl` in the command line. This will render to the `docs/build` directory. In there, simply click on the `index.html` file, which will open the (local version of the) website in your browser

is it always this complicate or you just set it up like this xD?

Reply:
@pat-alt i don't know why but some latex  equations do not get rendered in the html version, although quarto shows they are correctly written in the little square preview box on the left side of the screen.